samples/bot-all-cards/csharp/BotAllCards/Dialogs/MainDialog.cs

```diff
@@ -0,0 +1,156 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using BotAllCards.Cards;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Builder.Dialogs.Choices;
+using Microsoft.Bot.Schema;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.BotBuilderSamples
+{
+    public class MainDialog : ComponentDialog
+    {
+        protected string ConnectionName { get; }
+
+        public MainDialog(IConfiguration configuration)
+             : base(configuration["ConnectionName"])
+        {
+            ConnectionName = configuration["ConnectionName"];
+
+            // Define the main dialog and its related components.
+            AddDialog(new ChoicePrompt(nameof(ChoicePrompt)));
+            AddDialog(new WaterfallDialog(nameof(WaterfallDialog), new WaterfallStep[]
+            {
+                ShowAllCardAsync,
+                SelectCardAsync,
+            }));
+
+            // The initial child Dialog to run.
+            InitialDialogId = nameof(WaterfallDialog);
+        }
+
+        // 1. Prompts the user if the user is not in the middle of a dialog.
+        // 2. Re-prompts the user when an invalid input is received.
+        private async Task<DialogTurnResult> ShowAllCardAsync(WaterfallStepContext stepContext, CancellationToken cancellationToken)
+        {
+
```

Comment:
Remove extra empty line

Reply:
No empty line. Please refer the screen short.
![image](https://user-images.githubusercontent.com/117716554/214214703-b5f17ef4-51f0-43cc-82f7-eb9aab7410f8.png)